config/bundles/bluima.commons_0.1.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.1",
   "dependencies": [
     {
-      "value": "ch.epfl.bbp.nlp:bluima_commons:1.0.1"
+      "value": "ch.epfl.bbp.nlp:bluima_commons:1.0.2-SNAPSHOT"
```

Comment:
Yes, I did change the case of "configFile" parameter (https://github.com/BlueBrain/bluima/commit/28a63939c4b52ef57c2a5eb4db3615ddcbfcad88) as you asked. ;-)